src/modelinstance.cpp

```diff
@@ -533,14 +533,13 @@ Status ModelInstance::waitForLoaded(const uint waitForModelLoadedTimeoutMillisec
         SPDLOG_DEBUG("Model: {}, version: {} already loaded", getName(), getVersion());
         return StatusCode::OK;
     }
-    SPDLOG_INFO("Model: {} version: {} is still loading", getName(), getVersion());
```

Comment:
Removed as message is misleading (it happens when model is not in AVAILABLE state), and few lines later we have log message anyway.